src/components/Modal.js

```diff
@@ -93,13 +97,8 @@ export default function Modal({
             onBadgeClick={onBadgeClick}
           />
         </BadgeContainer>
-        <Title>Choose Price</Title>
-        <RangeBar
-          filters={filters}
-          active={filters[badgeKeys[10]]}
-          setFilters={setFilters}
-          onChange={handleChange}
-        />
+        <Title>Choose Price-Limit</Title>
+        <PriceBar />
```

Comment:
Tried to move the filters setFilter React State down to the PriceBar. The Price-Bar had its own  default state which was replaced by [filters, setFilters]. After that the Price Bar was not working in the Modal.

Reply:
one of the next steps, i.e. a branch, would be to add props and finish this component